curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects.english.md

```diff
@@ -10,29 +10,47 @@ forumTopicId: 301153
 At their most basic, objects are just collections of <dfn>key-value pairs</dfn>, or in other words, pieces of data mapped to unique identifiers that we call <dfn>properties</dfn> or <dfn>keys</dfn>. Let's take a look at a very simple example:
```

Comment:
```suggestion
At their most basic, objects are just collections of <dfn>key-value</dfn> pairs. In other words, pieces of data(<dfn>values</dfn>) mapped to unique identifiers that we call <dfn>properties</dfn> (<dfn>keys</dfn>). Let's take a look at a very simple example:
```

Reply:
Should be `its` and not `it's`.  The former is possessive and the latter would mean `It is`

Reply:
Thanks guys

@RandellDawson Could you amend please?